packages/renderer/src/lib/engine/Prune.svelte

```diff
@@ -74,6 +74,4 @@ async function prune(type: string) {
 }
 </script>
 
-<Button on:click="{() => openPruneDialog()}" title="Remove unused images" icon="{faTrash}">
-  Prune {type}
-</Button>
+<Button on:click="{() => openPruneDialog()}" title="Remove unused images" icon="{faTrash}">Prune</Button>
```

Comment:
Looks like we had an existing bug! Could you fix the tooltip here too: `title="Remove unused {type}"`?

Reply:
Solved